tests/cypress/support/commands.ts

```diff
@@ -262,22 +262,23 @@ Cypress.Commands.add('createRoleTemplate', ({roleType='Global', roleName, newUse
     cy.get('span[aria-label="Yes: Default role for new users"]').click();
   }
   
-  // Addition of resources and verbs linked to resources
-  if (resources) {
-    resources.forEach((resource, i) => {
-      // Iterate over Resource cells and add 1 resource
-      cy.get(`input.vs__search`).eq(2 * i + 1).click();
-      cy.contains(resource, { matchCase: false }).should("exist").click();
-      cy.clickButton("Add Resource");
-
-      if (verbs) {
-        verbs.forEach((verb) => {
-          cy.get(`input.vs__search`).eq(2 * i).click();
-          cy.get(`ul.vs__dropdown-menu > li`).contains(verb).should("exist").click();
-        });
-      }
-    });
-  }
+    // Addition of resources and verbs linked to resources
+    // Each resource is an object with 2 keys: resource and verbs
+    if (resources) {
```

Comment:
I would remove these `if`s - if it is empty array the `forEach`  won't iterate over it.